en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md

```diff
@@ -37,3 +37,113 @@ Follow the steps below to take a heap dump of the Adapter.
 ## Taking a heap dump of Enforcer
 
 Enforcer is implemented in Java and it is configured with required parameters to dump the heap information when out of memory errors happen. The dump can be collected from `/home/wso2/logs/heap-dump.hprof`
+
+## Taking a heap dump of Router
+
+Router uses the Envoy Proxy as the core component that does the traffic routing. The default Router which is used in you production environment is not supporting heap profiling and we have build a separate Router Docker image which enables heap profiling. Follow the steps below to take a heap profile of the Router.
```

Comment:
We have built

Reply:
fixed with https://github.com/wso2/docs-apim/pull/6067/commits/83e49a43e873b2b537d601f13df397e4b6e72c31